settings.php

```diff
@@ -29,6 +29,8 @@
     global $CFG;
     global $wirisconfigurationclass;
 
+    \core\notification::fetch();
```

Comment:
Can you please add a comment explaining why is this line here?

Reply:
updated!